rules-reviewed/azure/springboot/spring-boot-to-azure-password.windup.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0"?>
+<ruleset id="spring-boot-to-azure-password"
+         xmlns="http://windup.jboss.org/schema/jboss-ruleset"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            Check all properties and configuration files and all environment variables on the production deployment(s) for any secret strings and passwords.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-xml,3.0.0.Final" />
+        </dependencies>
+        <sourceTechnology id="springboot"/>
+        <targetTechnology id="azure-appservice"/>
+        <targetTechnology id="azure-aks"/>
+        <tag>password</tag>
+    </metadata>
+    <rules>
+        <rule id="spring-boot-to-azure-password-01000">
+            <when>
+                <filecontent filename="application{*}.{extensions}" pattern="{password}"/>
+            </when>
+            <perform>
+                <hint title="Password found in configuration file" category-id="potential" effort="3">
+                    <message>
+                        <![CDATA[
+                        Password found in configuration file.
+
+                        Consider using Azure Key Vault and/or parameter injection with application settings where possible.
+                        ]]>
+                    </message>
+                    <link title="Password found in configuration file" href="https://docs.microsoft.com/azure/developer/java/migration/migrate-spring-boot-to-app-service#passwords-and-secure-strings"/>
+                    <link title="Key Vault" href="https://docs.microsoft.com/azure/key-vault/"/>
+                    <link title="Use Key Vault references" href="https://docs.microsoft.com/azure/app-service/app-service-key-vault-references?tabs=azure-cli"/>
+                </hint>
+                <technology-tag level="IMPORTANT">Security</technology-tag>
+                <technology-tag level="INFORMATIONAL">Azure</technology-tag>
```

Comment:
We're used to not have the target as a `technology-tag` because it's not a technology "available" in the application yet since it's the target for the migration so my suggestion is to remove it.
```suggestion
```